mackerel-plugin-redash/lib/redash.go

```diff
@@ -93,11 +95,21 @@ func (p RedashPlugin) FetchMetrics() (map[string]interface{}, error) {
 // Do the plugin
 func Do() {
 	optURI := flag.String("uri", "http://localhost/api/admin/queries/tasks?api_key=hoge", "stats URI")
```

Comment:
With this p-r we should encourage users to use `-api-key` rather than `api_key` parameter in `-uri`.
So can you remove `?api_key=` from this default value?